source/operations/install-deploy-manage/upgrade-minio-tenant.rst

```diff
@@ -10,71 +10,107 @@ Upgrade a MinIO Tenant
    :local:
    :depth: 1
 
+
+The following procedures upgrade a single MinIO Tenant, using either ``kubectl`` or Helm.
+MinIO recommends you test upgrades in a lower environment such as a Dev or QA Tenant, before upgrading production Tenants.
+
 .. important::
 
    For Tenants using a MinIO Image older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` running with :ref:`AD/LDAP <minio-ldap-config-settings>` enabled, you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
    You must take the extra steps documented in the linked release as part of the upgrade procedure.
 
 .. _minio-upgrade-tenant-plugin:
+.. _minio-upgrade-tenant-kubectl:
 
-Upgrade the Tenant using the MinIO Kubernetes Plugin
-----------------------------------------------------
-
-The following procedure upgrades the MinIO Operator using the :mc:`kubectl minio tenant upgrade` command.
+Upgrade a Tenant using ``kubectl``
+----------------------------------
 
+The following procedure upgrades a MinIO Tenant using ``kubectl``.
 If you deployed the Tenant using :ref:`Helm <deploy-tenant-helm>`, use the :ref:`minio-upgrade-tenant-helm` procedure instead.
 
-This procedure *requires* a valid installation of the MinIO Kubernetes Operator and assumes the local host has a matching installation of the MinIO Kubernetes Operator and plugin.
-This procedure assumes the latest stable Operator version |operator-version-stable|.
+To upgrade a Tenant with ``kubectl``:
 
-.. include:: /includes/k8s/install-minio-kubectl-plugin.rst
+#. In a convenient directory, save the current Tenant configuration to a file using ``kubectl get``:
```

Comment:
This is *technically* optional if the user already deployed with Kustomize.

Maybe fork /tab this out.

One tab is "Existing Kustomize-Deployed Tenant"

The other is "Operator Console Deployed Tenant"

And for the second tab you can note that this also works for Tenants deployed using the deprecated `kubectl minio` plugin.

The main difference is in the former, step 1 is entirely optional. You just need to modify the `kustomization` files so the Tenant now deploys with an updated Image. They can do that by modifying the `Tenant.yaml` or adding a patch to `kustomization` in case they want to always keep the original image in record.

Reply:
Maybe keep the same, with a few words about "If you already deploy with Kustomize, use your existing file instead." If that works with only a few words, at any rate. 🤔

Reply:
yeaaaahhh tabs it is.